core/story-api/src/main/java/org/lgna/story/Orientation.java

```diff
@@ -63,7 +63,7 @@ public Orientation(Number x, Number y, Number z, Number w) {
     this(new OrthogonalMatrix3x3(new UnitQuaternion(x.doubleValue(), y.doubleValue(), z.doubleValue(), w.doubleValue())));
   }
 
```

Comment:
instead of exposing this, what if we added createUnitQuaternion and createEulerAngles as functions on Orientation?

Reply:
Expose?
This was unchanged and already public. I did make the constructor above (line 54) public, but that one just takes the internal matrix.
`UnitQuaternion` has a method `createOrthogonalMatrix3x3()` that could be used here instead of the wrapping constructor call, but it seems kind of pointless.

Reply:
github showed this comment in the most useless place. i'm referencing the change on line 66 where getInternal is made public

Reply:
Done. I am not crazy about the "create…" notation, but it is established.